lib/fs/basicfs_unix.go

```diff
@@ -52,6 +52,15 @@ func (f *BasicFilesystem) Hide(name string) error {
 	return err
 }
 
+func (f *BasicFilesystem) IsHidden(name string) (bool, error) {
+	fileInfo, err := f.Stat(name)
+	if err != nil {
+		return false, err
+	}
+	name = fileInfo.Name()
+	return name[0:1] == ".", nil
```

Comment:
name[0] == '.'

Reply:
I thought that name[0:1] wouldn't panic if name was an empty string. I must have misstyped something before, because now I checked and it panics. I will change it to yours, thanks.

Reply:
name[0] panics too for an empty string. And you also need to check for `name == "."`, which is not hidden.